src/i18n/en.js

```diff
@@ -204,25 +204,28 @@ by Doppler and how many don't. By tracking the user's journey you'll be able to
     visits_without_email: `Users without email`,
   },
   reports_partials_campaigns: {
-    campaign_name: `Campaign: `,
-    campaign_state: `Campaign State: `,
+    campaign_name: `Campaign name: `,
+    campaign_state: `Campaign status`,
```

Comment:
Maybe Name and Status should have the first letter in uppercase. In spanish is: Nombre de la  Campaña

Reply:
We only use capital letters for the start of the sentence and for Doppler particular objects like Campaigns, Lists, etc

Reply:
Talked with Mariano and he says its ok this works are in lowercase, Campaign Reports and Emails its ok in uppercase